CMakeLists.txt

```diff
@@ -475,6 +475,17 @@ if (NOT DEFINED MFEM_TIMER_TYPE)
   endif()
 endif()
 
+# ALE3D begin edit
+# Without this, CMake 3.21.1 (and 3.20.2) run into CMake Errors like the following:
+# CMake Error at config/cmake/modules/MfemCmakeUtilities.cmake:60 (add_library):
+#   Target "mfem" links to target "Threads::Threads" but the target was not
+#   found.  Perhaps a find_package() call is missing for an IMPORTED target, or
+#   an ALIAS target is missing?
+# Call Stack (most recent call first):
+#   CMakeLists.txt:474 (mfem_add_library)
+find_package(Threads REQUIRED)
```

Comment:
I wonder if there are cases in which we need to add this and others where we don't?  Do we have a good enough understanding of why these are coming up in your case, but not others to put a little logic in here?

Reply:
Not sure. These CMake things were in there before I started working on the implicit hydro stuff, I just brought over all the commits that had been made in ALE3D previously (as evidenced by the commits made 4 years ago by Jaime).